src/AddTodoForm.jsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+const AddTodoForm = () => {
+    return (
+        <div>
+            <h2>Add Todo</h2>
+            <form>
+                <label htmlFor="todoTitle">Title</label>
+                <input type="text" id="todoTitle" name="todoTitle"/>
```

Comment:
add a required attribute here 

```suggestion
<input type="text" id="todoTitle" name="todoTitle" required/>
```

Reply:
Thanks for pointing that out, Justice! Added it in!😊